index.html

```diff
@@ -90,10 +89,22 @@ <h5 class="card-title">Gaurav Burjwal</h5>
             <hr class="separator">
             <p class="card-text">I'm a Computer Science Undergraduate. Passionate about problem-solving through programming.Machine Learning Enthusiast.</p>
             <a href="https://github.com/gauravburjwal" class="button">GitHub Profile</a>
+
+          </div>
+        </div>
+      </div>
+      
+      <div class="col-12 col-md-6 col-lg-3 contributor">
+        <div class="card card__animate">
+          <img class="card-img-top img-fluid" data-src="./images/cops.png" alt="Card image cap">
```

Comment:
You didn't attached your image here.

Reply:
Oh ok, my bad, fixed it as per 3030655 @jmig5776